run-server.bat

```diff
@@ -9,7 +9,11 @@ SET ARG=%1
 SET ARG2=%2
 
 IF "%ARG%"=="/t" (
-  docker-compose -f %DEV% run web pytest
+     IF "%ARG2%"=="--cov" (
```

Comment:
I'd make it in a 'windows' manner, i.e. argument typed to CMD should be with forward slash, `/cov` or something like this.

Reply:
okej